client/src/app/components/SimpleSelect.tsx

```diff
@@ -40,6 +40,7 @@ export const SimpleSelect: React.FC<ISimpleSelectProps> = ({
 
   return (
     <Select
+      menuAppendTo="parent" // prevent menu from being clipped by modal edges
```

Comment:
Adding this overflow seems to be a much better experience, but not sure if it is the PF friendly way to accomplish this. 
```suggestion
      menuAppendTo="parent" // prevent menu from being clipped by modal edges
      style={{ overflowY: "auto", maxHeight: "200px" }}

```

Reply:
<img width="919" alt="Screenshot 2023-10-03 at 7 54 02 PM" src="https://github.com/konveyor/tackle2-ui/assets/11218376/1057e882-5542-458b-a9f7-c161c8a9c998">

Reply:
i added a maxHeight of 200

Reply:
I see the overflow now with the maxHeight set. LGTM other than the console.log